RealmSwift-swift3.0-dev/List.swift

```diff
@@ -31,7 +31,7 @@ public class ListBase: RLMListBase {
         return descriptionWithMaxDepth(depth: RLMDescriptionMaxDepth)
     }
 
-    @objc private func descriptionWithMaxDepth(depth: UInt) -> String {
+    @objc(descriptionWithMaxDepth:) private func descriptionWithMaxDepth(depth: UInt) -> String {
```

Comment:
The code that is calling this code is Objective-C++; neither `@selector(description:)` nor `@selector(description(maxDepth:))` are accepted.